parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -13,14 +13,12 @@ use codec::{Decode, Encode};
 use frame_support::{dispatch::DispatchResult, log, transactional};
 use frame_system::ensure_signed;
 use scale_info::TypeInfo;
-use snowbridge_beacon::{
-	BeaconBlock, BeaconHeader, Domain, ExecutionHeader, ForkData, PublicKey, Root, SigningData,
-	SyncAggregate, SyncCommittee,
-};
 use sp_core::H256;
 use sp_io::hashing::sha2_256;
 use sp_runtime::RuntimeDebug;
 use sp_std::prelude::*;
+use snowbridge_beacon::{SyncCommittee, BeaconHeader, SyncAggregate, ForkData, Root, Domain, PublicKey, SigningData, ExecutionHeader, BeaconBlock};
```

Comment:
I would name the beacon primitives crate to this for clarity in what the crate provides (primitives):

```suggestion
use snowbridge_beacon_primitives::{SyncCommittee, BeaconHeader, SyncAggregate, ForkData, Root, Domain, PublicKey, SigningData, ExecutionHeader, BeaconBlock};
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/653/commits/45db3ef5727c1fa56e7c5b0f12b387bce7f0cacd